src/utils/ops/repeatable_tasks.json

```diff
@@ -0,0 +1,140 @@
+[
```

Comment:
@idiom-bytes Please add your OPS tasks suggestions here or feel free to commit new tasks to this repo